### DISCUSSION TITLE
My account was hacked, Please, someone help me!!

### DISCUSSION THREAD

**Discussion by ghost** [community/community#141820] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

My account was hacked and changed to another email. I can't get any support here on the GitHub platform. My account has been hacked for over a year. My account is DuarteProg, but I can't recover it. What should I do?

I'm starting to feel desperate :(

I opened a ticket, but no one has responded, and it's been over a year.

**Comment by ghost** [comment]
My real account calls DuarteProg

**Comment by ebndev** [comment]
Hi @DuarteSenior,

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please try to open another ticket.

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
